apps/initializer/lib/initializer.ex

```diff
@@ -26,10 +26,16 @@ defmodule Initializer do
           Map.new(init_arg)
           |> Map.put(:supervisor_ref, supervisor_ref)
 
+        {:ok, state}
```

Comment:
handle_continue will never be called unless `init/1` returns `{:ok, state, {:continue, :something}}`

Reply:
OK. I misunderstood how that worked.

Reply:
Fixed